pkg/interface/src/views/apps/links/components/link-item.js

```diff
@@ -33,7 +33,7 @@ export const LinkItem = (props) => {
     ? <img src={props.avatar} height={36} width={36} className="dib" />
     : <Sigil ship={`~${author}`} size={36} color={'#' + props.color} />;
 
-  const baseUrl = props.baseUrl || `/~link/${resource}`;
+  const baseUrl = props.baseUrl || `/~404/${resource}`;
```

Comment:
Why not just `props.baseUrl`

Reply:
This is in the case that props.baseUrl is null or undefined. Better to explicitly redirect to /~404 imo